CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.22.0] - 2024-06-25
+
+### Added
+
+- CASMINST-6896: Add support for multiple GPG keys and update with CFS
+
```

Comment:
I am 99% sure that you add this to the "Unreleased" section in your PR.
Then in the develop->main PR someone will slap a version on it.

Reply:
That's the idea in theory, but that is under the idea that we may be making multiple merges into develop before we merge over to master. For a case like this where we know we're going to merge it over to master directly, this is okay.